snapcraft/plugins/autotools.py

```diff
@@ -113,18 +111,15 @@ def build(self):
                 self.run(['autoreconf', '-i'])
 
         configure_command = ['./configure']
-        make_install_command = ['make', 'install']
 
-        if self.install_via_destdir:
+        if self.options.make_install_var:
             # Use an empty prefix since we'll install via DESTDIR
             configure_command.append('--prefix=')
-            make_install_command.append('DESTDIR=' + self.installdir)
         else:
             configure_command.append('--prefix=' + self.installdir)
 
         self.run(configure_command + self.options.configflags)
-        self.run(['make', '-j{}'.format(self.parallel_build_count)])
-        self.run(make_install_command)
+        self.make()
```

Comment:
Forgive my density (I only just started the coffee IV), but how does this use #1017?